src/main/scripts/ncm-ncd

```diff
@@ -3,390 +3,35 @@
 # ${developer-info}
 # ${author-info}
 # ${build-info}
-#
-use lib '/usr/lib/perl';
```

Comment:
Where are you moving the libraries to?  RHEL5 and 6 had different installation paths for Perl modules.